### PR TITLE
Making some minor changes to maker-handbook.md

### PR DESCRIPTION
**Pull request by src4026**
Please check the suggested changes and commit them as you wish.

### CODE REVIEW

**Comment by a-blob** [comment]
The other two grammar fixes seems to make the sentence more confusing to me :/

File: maker-handbook.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ This is a detailed guide on how to create a map for the Minetest Capture the Fla
 * Open Minetest and select the Capture the Flag game.
   
   ![image](https://github.com/CTF-handbooks/map-maker-handbook/assets/88883098/5cce3993-a5f4-408a-9178-c177f5e6885a)
-* Create a new world. You can use any mapgen. You can use the v7 mapgen for generated terrain, or you can use the flat or singlenode mapgen if you want to make a map from scratch.
+* Create a new world. You can use any mapgen. You can use the v7 mapgen for generated terrain, the flat mapgen, or singlenode mapgen if you want to make a map from scratch.
```
Comment (a-blob) [inline maker-handbook.md]:
```suggestion
* Create a new world. You can use any mapgen. You can use the v7 mapgen for generated terrain, or you can use the flat or singlenode mapgen if you want to make a map from scratch.
```

File: maker-handbook.md [hunk 2]
```diff
@@ -42,7 +42,7 @@ This is a detailed guide on how to create a map for the Minetest Capture the Fla
 * To get started, grant yourself the "ctf_map_editor" by running `/grantme ctf_map_editor`. You can also use `/grantme all` to grant you "ctf_map_editor" and all the other privs that will be useful while making a map, such as fly, noclip, fast, etc...
 * You can hit Esc on your keyboard and click `Change Keys` to see your controls (such as how to fly, noclip, etc...).
 ### 5. Building the Map
-* It is your choice to build the borders first or the map first. For this example, we will build the map first.
+* It is your choice to build the borders or map first. For this example, we will build the map first.
```
Comment (a-blob) [inline maker-handbook.md:45]:
```suggestion
* It is your choice to build the borders first or the map first. For this example, we will build the map first.
```

**Review by a-blob — CHANGES_REQUESTED** [review]

**Comment by src4026** [comment]
> The other two grammar fixes seems to make the sentence more confusing to me :/

AFAIK the grammar ones are leveraging on the shared qualities of the words in those cases. I will check them once again :+1:

**Comment by src4026** [comment]
Oh wait, I missed a word in the first suggestion xD.

**Review by a-blob — APPROVED** [review]

**Comment by src4026** [comment]
~~Grammarly getting mad at s20.~~

**Pull request merged**